src/gateway/get-started/proxy-caching.md

```diff
@@ -1,73 +1,190 @@
+series from the beginning. 
 
-curl -i -X POST http://localhost:8001/plugins \
-  --data name=proxy-cache \
-  --data config.content_type="application/json" \
-  --data config.cache_ttl=30 \
-  --data config.strategy=memory
+Start with the introduction [Get Kong](/gateway/latest/get-started/get-kong), which includes
+tool prerequisites and instructions for running a local {{site.base_gateway}}.
 
-```
+Step two of the guide, [Services and Routes](/gateway/latest/get-started/services-and-routes),
+includes instructions for installing a mock service used throughout this series. 
 
-If configuration was successful, you will receive a `201` response code. The request you sent configured Proxy Caching for all `application/json` content, with a time to live (TTL) of 30 seconds. The final option `config.strategy=memory` specifies where the cache will be stored. You can read more about this option in the strategy section of the [Proxy Caching plugin](/hub/kong-inc/proxy-cache/) documentation. Because this request did not specify a route or a service, {{site.base_gateway}} has applied this configuration globally across all services and routes. The Proxy Caching plugin can also be configured at service-level, route-level, or consumer-level. You can read more about the other configurations and how to apply them in the [Proxy Caching plugin](/hub/kong-inc/proxy-cache/) page.
+If you haven't completed these steps already, complete them before proceeding.
 
-## Validate the configuration
+### Global proxy caching
 
-You can check that the Proxy Caching plugin is working by sending a `GET` request to the route that was created in the [configure services and routes](/gateway/latest/get-started/configure-services-and-routes) guide and examining the headers. If you did not follow the guide, edit the example to reflect your configuration. Send the following request once: 
+Installing the plugin globally means *every* proxy request to {{site.base_gateway}}
+will potentially be cached.
 
+1. **Enable proxy caching**
 
-```
-curl -i -X GET http://localhost:8000/mock/request
+   The Proxy Cachine plugin is installed by default on Kong Gateway, and can be enabled by 
+   sending a `POST` request to the plugins object on the Admin API:
+   
+   ```sh
+   curl -i -X POST http://localhost:8001/plugins \
+     --data "name=proxy-cache" \
+     --data "config.request_method=GET" \
+     --data "config.response_code=200" \
+     --data "config.content_type=application/json; charset=utf-8" \
+     --data "config.cache_ttl=30" \
+     --data "config.strategy=memory"
+   ```
+   
+   If configuration was successful, you will receive a `201` response code. 
+  
+   This Admin API request configured a Proxy Cache plugin for all `GET` requests that resulted
+   in response codes of `200` and *response* `Content-Type` headers that *equal*
+   `application/json; charset=utf-8`. `cache_ttl` instructed the plugin to flush values after 30 seconds.
```

Comment:
```suggestion
   `application/json; charset=utf-8`. `cache_ttl` instructed the plugin to flush values after 30 seconds.
```

Do you think it is necessary to say 'flush the cache'? instead of 'flush values'

Reply:
IMO saying "flush the cache" means flushing the entire cache (which is a different operation), but this defines the TTL of individual values